clients/broadcaster.go

```diff
@@ -49,16 +49,16 @@ type StreamAllocResponse struct {
 
 type EncodedProfile struct {
 	Name         string `json:"name,omitempty"`
-	Width        int32  `json:"width,omitempty"`
-	Height       int32  `json:"height,omitempty"`
-	Bitrate      int32  `json:"bitrate,omitempty"`
-	FPS          uint   `json:"fps,omitempty"`
-	FPSDen       uint   `json:"fpsDen,omitempty"`
+	Width        int64  `json:"width,omitempty"`
```

Comment:
We were using different types in different structs, so standardised on int64 / float64

Reply:
Go is much better than c++ when handling `int` and `uint` arithmetic !